.ci/apm-beats-update.groovy

```diff
@@ -99,3 +85,53 @@ pipeline {
   //   }
   // }
 }
+
+def beatsUpdate() {
+  def os = "linux"
+  def goRoot = "${env.WORKSPACE}/.gvm/versions/go${GO_VERSION}.${os}.amd64"
+
+  withEnv([
+    "HOME=${env.WORKSPACE}",
+    "GOPATH=${env.WORKSPACE}",
+    "GOROOT=${goRoot}",
+    "PATH=${env.WORKSPACE}/bin:${goRoot}/bin:${env.PATH}",
+    "MAGEFILE_CACHE=${env.WORKSPACE}/.magefile",
+  ]) {
+    dir("${BEATS_DIR}") {
+      sh(label: "Create branch localVersion", script: "git checkout -b localVersion")
+      sh(label: "Install Go ${GO_VERSION}", script: ".ci/scripts/install-go.sh")
+      sh(label: "Install Mage", script: "make mage")
```

Comment:
```suggestion
```

`mage` doesn't need to be installed if you remove the pythonEnv bit.

Reply:
I gonna try, but IIRC `make check` needs `mage pythonEnv` the needs `mage` :)

Reply:
removed